modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -30,6 +30,7 @@ NOTE: In the above snippet, the network settings must match the settings for the
 The IPv4 entries must go *before* the IPv6 entries.
 ====
 
+//why?
```

Comment:
@johnwilkins, I am curious why this is here - each release version has a separate copy of this file, so you don't need these if statements.